packages/test/test-end-to-end-tests/src/test/refreshSerializerLifeCycle.spec.ts

```diff
@@ -27,12 +25,14 @@ import {
 	timeoutAwait,
 	waitForContainerConnection,
 	type ChannelFactoryRegistry,
-	type ITestContainerConfig,
 	type ITestObjectProvider,
 } from "@fluidframework/test-utils/internal";
 
 import { wrapObjectAndOverride } from "../mocking.js";
 
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Can't you avoid this by re-exporting under `/offline/index.ts`?  Not sure if there's precedence either way.

Could also consider moving that utils file to another package, probably `@fluidframework/test-utils`? (tag it as internal)